services/queue/schemas/v1/object-credentials.yml

```diff
@@ -0,0 +1,29 @@
+$schema: "/schemas/common/metaschema.json#"
+title: "Object service Credentials"
+type: object
+description: |
+  Temporary credentials for access to the object service.
+
+  These credentials are used both to download artifacts from the object service
+  (`getArtifactContent`) and to upload artifacts (`createArtifact`).
+properties:
+  clientId:
+    type:       string
+    minLength:  1
+    description: |
+      The `clientId` for the temporary credentials.
+  accessToken:
+    type:       string
+    minLength:  1
+    description: |
+      The `accessToken` for the temporary credentials.
+  certificate:
+    type:       string
+    minLength:  1
+    description: |
+      The `certificate` for the temporary credentials.
```

Comment:
Do we have a schema already for temp credentials that we could reuse here? It feels like this could be a shared definition.

Reply:
We do - `task-credentials.yml`.  However, it contains comment specific to task credentials (around expiration and usage), and I wanted to add comments specific to object credentials, so I split them up.